app/views/propertyDetails/propertyDetailsSummary.scala.html

```diff
@@ -221,7 +224,8 @@ <h2 id="dates-of-liability-header" class="heading-medium">@messages("ated.proper
     </div>
 
     <div class="grid grid-1-6 cya-change">
-      <a href="@controllers.propertyDetails.routes.IsFullTaxPeriodController.editFromSummary(propertyDetails.id)" id="edit-dates-of-liablity-@index">@messages("ated.edit") <span class="visuallyhidden">@period.description</span>
+      <a href="@controllers.propertyDetails.routes.IsFullTaxPeriodController.editFromSummary(propertyDetails.id)" id="edit-dates-of-liablity-@index">@messages("ated.edit")
```

Comment:
Can you put @messages("ated.edit") on the next line with what's on 228 so it's clear that it will be displayed vs. visually hidden please? 

@messages("ated.edit") <span class="visuallyhidden">@messages((period.description).get)</span>